### ISSUE TITLE
[Accessibility][Keyboard Navigation - User Input]: 'All' and specific user input section is not accessible via Keyboard.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2047]
**User Experience:**
For keyboard users it is difficult to complete the task is if any control is not accessible via keyboard.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. Select 'User Input' link from left navigation.
3. Observe the issue.

**Actual Result:**
'All' and specific user input section is not accessible via Keyboard. On clicking using mouse, these controls are interactive as action is getting performed but its not accessible with keyboard.

**Note:**
Same issue is repro for 'Bot responses'.

**Expected Result:**
'All' and specific user input section should be accessible via Keyboard.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/CELA/Official_Microsoft_Accessibility_Standards/MAS%202.1.1%20%E2%80%93%20Keyboard.docx?d=w1f29acf9a3e7414fbd8d7e3598d877bb&csf=1&e=LSJeSF

**Attachment:**
[MAS2.1.1_Bot responses_.zip](https://github.com/microsoft/BotFramework-Composer/files/4239008/MAS2.1.1_Bot.responses_.zip)
[MAS2.1.1_User Input_Bot responses.zip](https://github.com/microsoft/BotFramework-Composer/files/4239009/MAS2.1.1_User.Input_Bot.responses.zip)

**Comment by corinagum** [comment]
![image](https://user-images.githubusercontent.com/14900841/75399187-7219c000-58b0-11ea-9c2b-39e8c319c2dc.png)

Focus does not move to green box area after tabbing from focus on 'Start bot' button

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.
**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
